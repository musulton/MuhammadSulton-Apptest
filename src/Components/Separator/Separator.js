import * as React from 'react';
import {View} from 'react-native';

import styles from './Separator.styles';

const Separator = () => <View style={styles.separator} />;

export default Separator;
