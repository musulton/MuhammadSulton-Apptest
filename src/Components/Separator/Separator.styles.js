import {StyleSheet} from 'react-native';

export default StyleSheet.create({
  separator: {
    height: 1.5,
    width: '100%',
    opacity: 0.3,
    backgroundColor: 'gray',
  },
});
