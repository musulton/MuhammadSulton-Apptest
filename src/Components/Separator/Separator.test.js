import * as React from 'react';
import {render, cleanup} from '@testing-library/react-native';

import Separator from './Separator';

describe('Separator', () => {
  afterEach(cleanup);

  it('should render separator correctly', () => {
    const {toJSON} = render(<Separator />);

    expect(toJSON()).toMatchSnapshot();
  });
});
