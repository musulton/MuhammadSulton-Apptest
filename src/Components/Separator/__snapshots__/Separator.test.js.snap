// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Separator should render separator correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "rgba(221, 221, 221, 0.3)",
      "height": 2,
      "width": "100%",
    }
  }
/>
`;
